blocks/bricks/__init__.py

```diff
@@ -464,6 +464,10 @@ def apply(self, input_):
             child_input = output
         return output
 
+    @apply.property('outputs')
```

Comment:
Adding multiple inputs support looks a little more complicated, though. At least, with my limited knowledge I don't have a clear idea of how to implement it. Could you give me some starting tips?